annotations/views/project_views.py

```diff
@@ -179,3 +195,35 @@ def list_projects(request):
         'projects': qs,
     }
     return render(request, template, context)
+
+
+@login_required
+def import_appellation(request, project_id):
+    context = {}
+    text_collection = TextCollection.objects.get(id = project_id)
+    if request.method == 'POST' and request.user == text_collection.ownedBy:
+        data = request.FILES['csv_file']
+        # read csv file
+        csv_file = csv_unicode.reader(data)
+        # Create Temp File.
+        name = 'tmp/'+ request.user.username + '_' + str(datetime.datetime.now())+'.csv'
+        tmp_file = default_storage.save(name, ContentFile(data.read()))
+        # write csv contents to temp file
+        with open(tmp_file, 'w') as f:
```

Comment:
why do you do that? doesn't this ```default_storage.save(name, ContentFile(data.read()))``` already do that, or why do you give it a contentfile?